pkg/combustion/kubernetes_test.go

```diff
@@ -161,15 +165,15 @@ func TestConfigureKubernetes_SuccessfulRKE2Server(t *testing.T) {
 	require.NoError(t, err)
 
 	contents := string(b)
-	assert.NotContains(t, contents, "export INSTALL_RKE2_TYPE=server",
-		"INSTALL_RKE2_TYPE is set when the definition file does not explicitly set it")
 	assert.Contains(t, contents, "cp server-images/* /var/lib/rancher/rke2/agent/images/")
-	assert.Contains(t, contents, "cp rke2_config.yaml /etc/rancher/rke2/config.yaml")
+	assert.Contains(t, contents, "cp server.yaml /etc/rancher/rke2/config.yaml")
+	assert.Contains(t, contents, "cp rke2-vip.yaml /var/lib/rancher/rke2/server/manifests/rke2-vip.yaml")
```

Comment:
As before, I suspect this manifest will be identical for k3s too, so perhaps it makes sense to rename it, although we can always do that when we land k3s work.